### PR TITLE
Localization update for 3.24

### PR DESCRIPTION
**Pull request by mikeger**
_This field was intentionally left blank._

### CODE REVIEW

File: Wire-iOS/Resources/de.lproj/Localizable.strings [hunk 1]
```diff
@@ -412,10 +434,10 @@
 "content.system.conversation.with_name.participants" = "mit";
 
 "content.system.message_timer_changes" = "%@ hat temporäre Nachrichten auf %@ gestellt";
-"content.system.message_timer_changes-you" = "Du hast temporäre Nachrichten auf %@ gestellt";
+"content.system.message_timer_changes-you" = "%@ hat temporäre Nachrichten auf %@ gestellt";
```
Comment (mikeger) [inline Wire-iOS/Resources/de.lproj/Localizable.strings]: Not sure about those changes, seems wrong to me.

File: Wire-iOS/Resources/de.lproj/Localizable.strings [hunk 2]
```diff
@@ -412,10 +434,10 @@
...
 
 "content.system.message_timer_off" = "%@ hat temporäre Nachrichten ausgeschaltet";
-"content.system.message_timer_off-you" = "Du hast temporäre Nachrichten ausgeschaltet";
+"content.system.message_timer_off-you" = "%@ hat temporäre Nachrichten ausgeschaltet";
```
Comment (mikeger) [inline Wire-iOS/Resources/de.lproj/Localizable.strings]: Also off.

File: Wire-iOS/Resources/de.lproj/Localizable.strings [hunk 3]
```diff
@@ -472,18 +495,19 @@
 "content.message.copy" = "Kopieren";
 "content.message.download" = "Download";
 
+// Reply
+"content.message.reply" = "Antworten";
+"content.message.reply.original_timestamp.date" = "Original message from %@";
+"content.message.reply.original_timestamp.time" = "Original message from %@";
```
Comment (mikeger) [inline Wire-iOS/Resources/de.lproj/Localizable.strings]: Also off.

Comment (typfel) [reply Wire-iOS/Resources/de.lproj/Localizable.strings]: Andy will fix this

Comment (typfel) [reply Wire-iOS/Resources/de.lproj/Localizable.strings]: @billypchan  fixed it :)

Comment (mikeger) [reply Wire-iOS/Resources/de.lproj/Localizable.strings]: In crowdin? Somebory revoked my access there :suspect:

File: Wire-iOS/Resources/fr.lproj/Localizable.strings [hunk 4]
```diff
@@ -472,6 +495,12 @@
 "content.message.copy" = "Copier";
 "content.message.download" = "Télécharger";
 
+// Reply
+"content.message.reply" = "Répondre";
+"content.message.reply.original_timestamp.date" = "Original message from %@";
```
Comment (alexaubry) [inline Wire-iOS/Resources/fr.lproj/Localizable.strings]: I need to update this.

Comment (alexaubry) [reply Wire-iOS/Resources/fr.lproj/Localizable.strings]: Done

Comment (typfel) [reply Wire-iOS/Resources/fr.lproj/Localizable.strings]: Alright, I'll close this PR and re-run the Crowdin job.

File: Wire-iOS/Resources/de.lproj/Localizable.strings (hunk 1 above)
Comment (typfel) [reply Wire-iOS/Resources/de.lproj/Localizable.strings]: Right I'll revert them on crowd in

Comment (typfel) [reply Wire-iOS/Resources/de.lproj/Localizable.strings]: Hmm not so easy, the code depend on this atm.

File: Wire-iOS/Resources/de.lproj/Localizable.strings (hunk 3 above)
Comment (billypchan) [reply Wire-iOS/Resources/de.lproj/Localizable.strings]: I had submitted the change and waiting for approval.

**Pull request closed without merging**